### ISSUE TITLE
error compiling draw example on linux

### ISSUE THREAD

**Issue by brinoausrino** [nannou-org/nannou#707]
Hey,

when I tried compiling the draw example, this error occurs

```error[E0282]: type annotations needed
  --> /home/brian/.cargo/registry/src/github.com-1ecc6299db9ec823/sample-0.9.1/src/window.rs:73:41
   |
73 |         let v = phase.to_float_sample().to_sample() * PI_2;
   |                                         ^^^^^^^^^ cannot infer type for type parameter `S` declared on the associated function `to_sample`
   |
help: consider specifying the type argument in the method call
   |
73 |         let v = phase.to_float_sample().to_sample::<S>() * PI_2;
   |     
 ```

Maybe someone has any idea (sry I'm quite new to rust)?

**Comment by rjpgt** [comment]
I guess this is the same issue as [#699](https://github.com/nannou-org/nannou/issues/699). The fix is to change the `audrey` version to `0.3` in the Cargo.toml file in the examples folder.

**Comment by brinoausrino** [comment]
Works, thank you!
